lighthouse-core/scripts/compare-timings.js

```diff
@@ -89,19 +103,46 @@ function round(value) {
   return Math.round(value * 10) / 10;
 }
 
-function collect() {
+function gather() {
   const outputDir = dir(argv.name);
-  if (!fs.existsSync(ROOT_OUTPUT_DIR)) fs.mkdirSync(ROOT_OUTPUT_DIR);
+  mkdirp.sync(ROOT_OUTPUT_DIR);
+  // Don't overwrite a previous collection
   if (fs.existsSync(outputDir)) throw new Error(`folder already exists: ${outputDir}`);
   fs.mkdirSync(outputDir);
 
   for (const url of argv.urls) {
     for (let i = 0; i < argv.n; i++) {
+      const gatherDir = `${outputDir}/${urlToFolder(url)}/${i}/`;
+      mkdirp.sync(gatherDir);
+      // TODO include multiple urls too.
+      const progressBar = new Array(Math.round(i * 40/ argv.n)).fill('▄').join('').padEnd(40);
+      console.log(`Gathering:   `, '|', progressBar, '|')
```

Comment:
this would be so much better if you grabbed the logger from the lantern collect scripts